CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## [v0.8.0](https://github.com/delta-io/delta-kernel-rs/tree/v0.8.0/) (2025-03-04)
+
+[Full Changelog](https://github.com/delta-io/delta-kernel-rs/compare/v0.7.0...v0.8.0)
+
+### 🏗️ Breaking changes
+
+1. ffi: `get_partition_column_count` and `get_partition_columns` now take a `Snapshot` instead of a
+   `Scan` ([#697])
+2. ffi: expression visitor callback `visit_literal_decimal` now takes `i64` for the upper half of a 128-bit int value  ([#724])
+3. - `DefaultJsonHandler::with_readahead()` renamed to `DefaultJsonHandler::with_buffer_size()` ([#711])
+4. DefaultJsonHandler's defaults changed:
+  - default buffer size: 10 => 1000 requests/files
+  - default batch size: 1024 => 1000 rows
+5. Bump MSRV to rustc 1.81 ([#725])
```

Comment:
Should [this](https://github.com/delta-io/delta-kernel-rs/pull/717) be included too?

Reply:
nope! included in 0.7.0 release